### ISSUE TITLE
Stylistic refinements

### ISSUE THREAD

**Issue by 1j01** [andywer/postcss-debug#20]
The chevrons on the expandable/collapsible sections feel wonky and backwards.
When chevrons are on the right side it's acceptable to have them face face down when collapsed and up when expanded because if they pointed right it would look like they were links to external content, but when they're on the left the standard is for chevrons to face right when collapsed and face down when expanded, sort of "towards" the content.

The dark theme looks a bit unpolished, especially with the light gray bottom borders of the snapshot accordion headers. Also the search inputs are light, and the "<kbd>Shift</kbd>+<kbd>D</kbd>" is light-on-light.

The layout changes based on how long the file names are and which files are displayed, with no good minimum/maximum widths. This makes searching feel not as nice.

The shadow on the selected file tab gets cut off because of the overflow rule used for scrolling. This can probably be fixed this by moving some of the spacing of the layout into the padding of the file list.

Vertical scrollbars appeared on some file tabs at some point. I don't know how to reproduce this.

The header contents aren't perfectly centered. This can be done fairly easily with flexbox.

**Comment by andywer** [comment]
Yep, good points 👍 I would gently pass this to @Gaserd if he has got some time 😉

**Comment by Gaserd** [comment]
@andywer @1j01 Hmmm... good points. I will think how to do it better! THX!
